website/docs/r/cloud_shell_account_settings.html.markdown

```diff
@@ -0,0 +1,94 @@
+  	key = "eu-de"
+  }
+  regions {
+  	enabled = true
+  	key = "jp-tok"
+  }
+  regions {
+  	enabled = true
+  	key = "us-south"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `account_id` - (Required, Forces new resource, string) The account ID in which the account settings belong to.
+* `default_enable_new_features` - (Optional, bool) You can choose which Cloud Shell features are available in the account and whether any new features are enabled as they become available. The feature settings apply only to the enabled Cloud Shell locations.
+* `default_enable_new_regions` - (Optional, bool) Set whether Cloud Shell is enabled in a specific location for the account. The location determines where user and session data are stored. By default, users are routed to the nearest available location.
+* `enabled` - (Optional, bool) When enabled, Cloud Shell is available to all users in the account.
+* `features` - (Optional, List) List of Cloud Shell features.
+  * `enabled` - (Optional, bool) State of the feature.
+  * `key` - (Optional, string) Name of the feature.
+* `regions` - (Optional, List) List of Cloud Shell region settings.
+  * `enabled` - (Optional, bool) State of the region.
+  * `key` - (Optional, string) Name of the region.
+* `rev` - (Optional, string) Unique revision number for the settings object.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The unique identifier of the cloud_shell_account_settings.
+* `created_at` - Creation timestamp in Unix epoch time.
+* `created_by` - IAM ID of creator.
+* `default_enable_new_features` - You can choose which Cloud Shell features are available in the account and whether any new features are enabled as they become available. The feature settings apply only to the enabled Cloud Shell locations.
+* `default_enable_new_regions` - Set whether Cloud Shell is enabled in a specific location for the account. The location determines where user and session data are stored. By default, users are routed to the nearest available location.
+* `enabled` - When enabled, Cloud Shell is available to all users in the account.
+* `features` - List of Cloud Shell features. Nested `features` blocks have the following structure:
+	* `enabled` - State of the feature.
+	* `key` - Name of the feature.
+* `regions` - List of Cloud Shell region settings. Nested `regions` blocks have the following structure:
+	* `enabled` - State of the region.
+	* `key` - Name of the region.
+* `rev` - Unique revision number for the settings object.
+* `type` - Type of api response object.
+* `updated_at` - Timestamp of last update in Unix epoch time.
+* `updated_by` - IAM ID of last updater.
+
+## Import
+
+You can import the `cloud_shell_account_settings` resource by using `account_id`.
+The `account_id` property can be formed from `account_id`, and `account_id` in the following format:
+
+```
+<account_id>/<account_id>
```

Comment:
i think here we need only one account_id after u changed in resource

Reply:
done